frontstage/controllers/party_controller.py

```diff
@@ -145,7 +139,8 @@ def resend_verification_email(party_id):
         response.raise_for_status()
     except requests.exceptions.HTTPError:
         logger.exception('Re-sending of verification email failed', party_id=party_id)
-        raise ApiError(logger, response, log_level='exception', message='Re-sending of verification email failed')
+        logger.error('Re-sending of verification email failed')
```

Comment:
This isn't needed.  The logger.exception line above does basically the same thing